nca/NetworkConfig/TopologyObjectsFinder.py

```diff
@@ -194,6 +203,8 @@ def _add_networkset_from_yaml(self, networkset_object):
         for cidr in cidrs:
             ipb.add_cidr(cidr)
         self._add_peer(ipb)
+        if ExplTracker().is_active():
+            ExplTracker().add_item(networkset_object.path, networkset_object.line_number, ipb.full_name())
```

Comment:
I think it may be redundant in the output (as well as html output) to specify anything about ip-blocks, even if the user may click on them.. (the explanation for connections between ip-block and workload may refer to policies involving only that workload).
@zivnevo  do you have any preference here?

Reply:
OK, so a connection involving ipblocks does not have to include both src and dst configurations, but we look at it as a whole block of configurations. It may have 2 parts or one, but we look at it as one list of configurations explaining the connections. Thus, removing explanations for ipblocks.